### ISSUE TITLE
psth function doesn't work with published DANDIset

### ISSUE THREAD

**Issue by cechava** [catalystneuro/distributed-coding-matnwb#12]
using [Steinmetz et al., 2019 DANDIset](https://gui.dandiarchive.org/#/dandiset/000017/draft)

The snippet below reproduces the error
```matlab

DANDIset_folder = '~/DANDIsets/000017/';
filepath = [DANDIset_folder 'sub-Moniz/sub-Moniz_ses-20170515T120000.nwb'];
% read in file
nwb_file = nwbRead(filepath);

psth(...
   nwb_file, ...
   unit_id = 5, ...
   group_by = 'visual_stimulus_left_contrast', ...
   before_time = -1, ...
   after_time = 1, ...
   n_bins = 50, ...
   psth_plot_option = 'gaussian', ...
   std = 0.03);

Error using cat
Dimensions of arrays being concatenated are not consistent.

Error in cell2mat (line 75)
            m{n} = cat(2,c{n,:});

Error in psth (line 162)
        group_spike_times = cell2mat(groupby_spike_times{u});

```

**Comment by bendichter** [comment]
This seems like it could be an array transpose issue

**Comment by cechava** [comment]
indeed, that's what it is

**Comment by cechava** [comment]
fixed with merge of PR #14
